js/ui/windowManager.js

```diff
@@ -585,6 +585,99 @@ const TilePreview = new Lang.Class({
     }
 });
 
+/**
+ * The direction a window should rotate in
+ */
+const RotationDirection = {
```

Comment:
8e41fc - I went with `GtkDirectionType` because it was the same number of characters :)